src/fixate/drivers/pps/bk_178x.py

```diff
@@ -355,21 +358,6 @@ def identify(self, as_string=False):
                 ret_val += "{}: {},".format(key, value)
         return ret_val
 
-
-if __name__ == "__main__":
-    from fixate.drivers import pps
-
-    mypps = pps.open(restrictions={"baud_rates": [9600]})
-    print(mypps.identify(as_string=True))
-    mypps.remote = True
-    mypps.voltage = 12
-    mypps.output_ch1 = True
-    mypps.output_ch1 = False
-    mypps.voltage = 5.5
-    mypps.output_ch1 = True
-    mypps.output_ch1 = False
-    mypps.voltage = 24
-    mypps.output_ch1 = True
-    mypps.output_ch1 = False
-
-    mypps.output_ch1 = False
+    def get_identity(self) -> str:
+        # TODO: this won't be formatted how we want
+        return self.identify(as_string=True)
```

Comment:
Note sure I follow, but I assume you mean to come back to this and fix the formatting?

Reply:
The self.identify function doesn't return a comma delimited string and includes the keys. Will make it consistent later.